test/e2e/flowaggregator_test.go

```diff
@@ -427,6 +426,26 @@ func testHelper(t *testing.T, data *TestData, podAIPs, podBIPs, podCIPs, podDIPs
 		}
 	})
 
+	// ToExternalFlows tests the export of IPFIX flow records when a source Pod
+	// sends traffic to an external IP
+	t.Run("ToExternalFlows", func(t *testing.T) {
+		// Creating a agnhost server as a host network pod
+		serverPodPort := int32(80)
+		_, serverIPs, cleanupFunc := createAndWaitForPod(t, data, func(name string, nodeName string) error {
+			return data.createServerPod(name, "", serverPodPort, false, true)
+		}, "test-server-", "")
+		defer cleanupFunc()
+
+		clientName, clientIPs, cleanupFunc := createAndWaitForPod(t, data, data.createBusyboxPodOnNode, "test-client-", nodeName(0))
+		defer cleanupFunc()
+
+		if !isIPv6 {
+			checkRecordsForToExternalFlows(t, data, clientName, clientIPs.ipv4.String(), serverIPs.ipv4.String(), serverPodPort, isIPv6)
```

Comment:
Despite there's an `isIPv6`, do you think we should check if `clientIPs.ipv4`, `serverIPs.ipv4` exist or not to avoid potential invalid memory issue? Same for other usages in other tests.

Reply:
Just checked, I suppose this [line](https://github.com/antrea-io/antrea/blob/b28b58ea20d627bbb2a28fba802d360946d72fa9/test/e2e/framework.go#L1102) in `podWaitForIPs` will report error if the ip doesn't exist?

Reply:
But it checks when hostnetwork is false and your server is a hostnetwork one, right?
https://github.com/antrea-io/antrea/blob/b28b58ea20d627bbb2a28fba802d360946d72fa9/test/e2e/framework.go#L1100

Reply:
yes that's correct. I didn't realized that at first. I think it might reports error in [line](https://github.com/antrea-io/antrea/blob/b28b58ea20d627bbb2a28fba802d360946d72fa9/test/e2e/networkpolicy_test.go#L973) in this case, but I'm not sure. I need to do more tests. Thanks a lot for pointing it out.